frontend/public/components/edit-yaml.jsx

```diff
@@ -61,13 +63,17 @@ export const EditYAML_ = connect(stateToProps)(
           sampleObj: props.sampleObj,
           fileUpload: props.fileUpload,
           showSidebar: !!props.create,
+          data: undefined,
```

Comment:
It's generally better to default to `null` when setting it explicitly (and that's what's used just above).